src/services/broadcast-service.js

```diff
@@ -129,7 +129,7 @@ class BroadcastService {
       );
       logger.debug(`send1to1Messge returns=${uMessage}`);
       reply.pending = 'Broadcast message in process of being sent to list of users';
-      reply.message = messageToSend;
+      reply.message = this.user.message;
```

Comment:
This is not a valid change, it should be messageToSend.  This value was generated based on the message, and the acknowledgement setting, etc...

Reply:
is something else readying this full message value (message, ack, and sentby) being returned by the broadcast service.sendMessage? My frontend needs the raw message?

Reply:
I scanned the codebase, it doesn't look like anything else reads the .message value from broadcast message so it should be safe to touch

Reply:
I added rawMessage, and changed message back to messageToSend